src/test/scala/com/fulcrumgenomics/bam/ClipBamTest.scala

```diff
@@ -524,4 +524,16 @@ class ClipBamTest extends UnitSpec with ErrorLogLevel with OptionValues {
       }
     }
   }
+
+  it should "clip FR reads that extend past the mate" in {
```

Comment:
`SamRecordClipper` has nice tests, so just wrote the minimal test here to ensure it works in `ClipBam`.